src/awsexplorer/awsExplorer.ts

```diff
@@ -18,10 +18,10 @@ import { RegionNode } from './regionNode'
 
 const ROOT_NODE_SIGN_IN = new AWSCommandTreeNode(
     undefined,
-    localize('AWS.explorerNode.signIn', 'Connect to AWS...'),
+    localize('AWS.explorerNode.connecting.label', 'Connecting...'),
```

Comment:
Do you intend for this node to remain clickable (which runs the aws.login command)? If not, we should define two separate nodes, and display the correct node as appropriate.

Reply:
I was hoping no one would ask for that because it's low-risk to keep it as-is, and doesn't seem worth the extra code.

Reply:
`getRootNodes()` defines the nodes "passively". Working around that looks awkward. So mutating the `label` looks most reasonable.

Reply:
You can manage a root node internal to the class and call `refresh()` to trigger updates.

Reply:
right, but how is that an improvement?

Reply:
`const ROOT_NODE_SIGN_IN` is intended as an immutable definition, and the change now uses it to store state. These types of changes tend to cause future maintenance issues.

Reply:
won't the "managed root node" need to be mutable itself? I.e. assigning a shared variable.